site/_docs/themes.md

```diff
@@ -94,3 +94,5 @@ Themes are published via [RubyGems.org](https://rubygems.org). You'll need a Rub
 2. Next, push your packaged theme up to the RubyGems service, by running the following command, again replacing `my-awesome-jekyll-theme` with the name of your theme:
 
         gem push my-awesome-jekyll-theme-*.gem
+
+3. To release a new version of your theme, simply update the version number in the gemspec file, ( `my-awesome-jekyll-theme.gemspec` in this example ), following the principles of [Semantic Versioning](http://semver.org/) and then repeat Steps 1 & 2 above.
```

Comment:
let's be specific like in the above steps.
To release 
* bump version in `spec.version       = "0.1.0"` in your `.gemspec`